aiohttp/web_urldispatcher.py

```diff
@@ -412,7 +411,7 @@ def get_info(self) -> _InfoDict:
         return {"path": self._path}
 
     def url_for(self) -> URL:  # type: ignore[override]
-        return URL.build(path=self._path, encoded=True)
+        return URL.build(path=_requote_path(self._path), encoded=True)
```

Comment:
I'm still not entirely clear what this change relates to.

Reply:
probably not needed

Reply:
removed